applications/doodle/src/doodle_exe.erl

```diff
@@ -253,7 +253,7 @@ init([Call]) ->
     process_flag('trap_exit', 'true'),
     CallId = whapps_call:call_id(Call),
     put('callid', CallId),
-    self() ! 'initialize',
+    gen_listener:cast(self(), 'initialize'),
```

Comment:
Yes: Sorry I was updating commit message as you commented. In case you didn't see:

I see that the 'initialize' is caught in doodle_exec:handle_info vs handle_cast and not quite the same then ensures, but this seems like a good place to ask why. Why doesn't the flow match callflow?